### ISSUE TITLE
Won't build...

### ISSUE THREAD

**Issue by manofstick** [enkerr/NewRepo#1]
Hi Rom,

Not sure if you have committed everything correctly to the repository (or even if it's mean to do anything as a project, or just as a general sample of code?) Anyway, if I pull the repo and try to build within Visual Studio it just doesn't - initially because files are blank, where it errors, so I just removed those files. Then the VolatilitySimulations project references the SqlFunctions project which wasn't building, so I removed that.  Then I got some more errors.

Anyway, maybe you're just battling with git? Anyway, let me know. But I'll just have a peruse of the code without it being built.

**Comment by enkerr** [comment]
Hi Paul,

Thanks for the message. This definitely won’t build! I just meant to share a sample or excerpt to illustrate my personal approach to problem solving, algorithms, and code organization, and what knowledge I have of F# (all at the unadvanced level I function at). The full solution depends on pulling data from my personally created SQL DB. I could potentially set the computations to run from free web-sourced data directly (eg, AlphaVantage). But that would take a couple days. Or I could manually shoehorn in some fake data to run against. That would be pretty simple. Let me know if that would be useful. But I originally was just going to paste one module as text in an email to Ewa... .

Thanks again for looking,
Rom

**Comment by manofstick** [comment]
No worries :-)

If it's not too painful then some dummy data and stubbed out functions that write back results, etc. would be nice; but I guess if the goal is just to have a general perusal of your efforts then I could live without it - so don't kill yourself try in making it happen if there is a lot of work.

**Comment by manofstick** [comment]
No, going to have to look further if you want to assassinate me :-)

(It was probably David, our db and market data guru. I do ride my bike down that way, but incognito!)
